src/CollectionsAndGenerics/Program.cs

```diff
@@ -0,0 +1,96 @@
+﻿// <copyright file="Program.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace CollectionsAndGenerics
+{
+    using System.Text.RegularExpressions;
+
+    /// <summary>
+    /// Program class - Asks for user choice of operation to perform with collections
+    /// </summary>
+    public partial class Program
+    {
+        /// <summary>
+        /// CollectionsAvailable - All collections provided to the user
+        /// </summary>
+        private enum CollectionsAvailable
+        {
+            Lists = 1,
+            Stacks = 2,
+            Queues = 3,
+            Dictionary = 4,
+            Exit = 5,
+        }
+
+        /// <summary>
+        /// Main function - Entry point and starts the application
+        /// </summary>
+        /// <param name="args">String array of arguments</param>
+        public static void Main(string[] args)
+        {
+            bool exit = false;
+            do
+            {
+                Console.WriteLine("COLLECTIONS");
+                foreach (CollectionsAvailable collections in Enum.GetValues(typeof(CollectionsAvailable)))
+                {
+                    Console.WriteLine($"{(int)collections}. {collections.ToString()}");
+                }
+
+                Console.WriteLine("Specify the type of collection you are interested to work with(Eg 1 for Lists): ");
+                int userChoice = ValidateInput.Program.GetValidUserChoice(Enum.GetNames(typeof(CollectionsAvailable)).Length);
+                switch (userChoice)
+                {
+                    case (int)CollectionsAvailable.Lists:
+                        ListHandler<string> listHandlerObject = new ListHandler<string>();
```

Comment:
Can we simplify this as 
```suggestion
                        ListHandler<string> listHandlerObject = new ();
```

Same applies for rest of the declarations as well

Reply:
Simplified object creation